modules/jfx.incubator.input/src/main/java/jfx/incubator/scene/control/input/InputMap.java

```diff
@@ -276,11 +280,15 @@ public void registerKey(KeyBinding k, FunctionTag tag) {
     }
 
     /**
-     * Unbinds the specified key binding.
+     * Unregister the specified key binding.
+     * Calling this method will disable any mappings made with
+     * {@link #register(KeyBinding, Runnable)},
+     * {@link #registerKey(KeyBinding, FunctionTag)},
+     * or registered by the skin.
```

Comment:
The "or registered by the skin" part seems doesn't seem right to me. Is that really what happens? And if so, why is it the right thing to do?

Reply:
this call disables the key binding, regardless of who registered it.

the use case example is Eclipse's "Unbind Command":

![Screenshot 2024-12-17 at 11 32 33](https://github.com/user-attachments/assets/cb39eb4c-969b-4967-9c10-9acd133c33b9)